lib/appsignal/rack/hanami_instrumentation.rb

```diff
@@ -0,0 +1,54 @@
+require "rack"
+
+module Appsignal
+  # @api private
```

Comment:
can you move this comment to the `HanamiInstrumentation` class?
and can you add the usage as a comment as well?

```ruby
    # @example
    #   class MyHanamiApp
    #     # Add this middleware to your app
    #     use Appsignal::Rack::HanamiInstrumentation
    #   end
```

or something like that